features/steps/thens/alignment.py

```diff
@@ -63,8 +80,8 @@ def step_impl(context, inst, ifc_rep_criteria, ifc_layout_criteria):
                                                 severity=OutcomeSeverity.ERROR)
 
             case ("IfcGradientCurve", "presence of IfcAlignmentVertical"):
-                if align.gradient_curves is not None:
-                    if align.verticals is None:
+                if align.gradient_curve is not None:
+                    if align.vertical is None:
                         yield ValidationOutcome(inst=inst, expected=ifc_layout_criteria, observed=None,
```

Comment:
And depending on whether `existence` equals `presence` or `absence` we'll say:

In case of presence, we expected {'entity':'IfcGradientCurve'} 
With the new function that parses the output into json that Raphael can work with, that means simply returning `expected=entities` and `observed=None`

In the case of absence, it'll be vice versa. We expected None, but observed the presence of an entity (or one of the entities)

I guess this will be good enough for the first version.